src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+]])
+
+-- }}} iproto configuration
+
+-- {{{ labels configuration
+
+M['labels'] = format_text([[
+    The `labels` section allows adding custom attributes to the configuration.
+    Attributes must be key: value pairs with string keys and values.
+]])
+
+M['labels.*'] = 'A value of the label with the specified name.'
+
+-- }}} labels configuration
+
+-- {{{ log configuration
+
+M['log'] = format_text([[
+    The `log` section defines configuration parameters related to logging.
+    To handle logging in your application, use the log module.
+]])
+
+M['log.file'] = format_text([[
+    Specify a file for logs destination. To write logs to a file, you need
+    to set `log.to` to file. Otherwise, `log.file` is ignored.
+]])
+
+M['log.format'] = format_text([[
+    Specify a format that is used for a log entry. The following formats
+    are supported:
+
+    - `plain`: a log entry is formatted as plain text.
+    - `json`: a log entry is formatted as JSON and includes additional fields.
+]])
+
+M['log.level'] = format_text([[
+    Specify the level of detail logs have. There are the following levels:
+
+    - 0: `fatal`
+    - 1: `syserror`
+    - 2: `error`
+    - 3: `crit`
+    - 4: `warn`
+    - 5: `info`
+    - 6: `verbose`
+    - 7: `debug`
+
+    By setting log.level, you can enable logging of all events with
+    severities above or equal to the given level.
+]])
+
+M['log.modules'] = format_text([[
+    Configure the specified log levels (`log.level`) for different modules.
+
+    You can specify a logging level for the following module types:
+
+    - Modules (files) that use the default logger. Example: Set log levels for
+    files that use the default logger.
+    - Modules that use custom loggers created using the `log.new()` function.
+    Example: Set log levels for modules that use custom loggers.
+    - The tarantool module that enables you to configure the logging level
+    for Tarantool core messages. Specifically, it configures the logging level
+    for messages logged from non-Lua code, including C modules. Example: Set
+    a log level for C modules.
+]])
+
+M['log.modules.*'] = format_text([[
+    Path to module.
+
+    For example: you have module placed by the following path:
+    `test/module.lua.` To configure logging levels, you need to
+    provide module names corresponding to paths to these modules:
+    ``` test.module: 'verbose' ```.
+]])
+
+M['log.nonblock'] = format_text([[
+    Specify the logging behavior if the system is not ready to write.
+    If set to `true`, Tarantool does not block during logging if the system
+    is non-writable and writes a message instead. Using this value may
+    improve logging performance at the cost of losing some log messages.
+]])
+
+M['log.pipe'] = format_text([[
+    Start a program and write logs to its standard input (`stdin`). To send
+    logs to a program's standard input, you need to set `log.to` to `pipe`.
+]])
+
+M['log.syslog'] = 'Syslog configurations params.'
```

Comment:
I would say 'To write logs to syslog, you need to set `log.to` to `syslog`' here too.

Reply:
BTW, I would use 'parameters' in a prose text, because 'params' feels as a jargon.

Reply:
Fixed.
```lua
I['log.syslog'] = format_text([[
    Syslog configurations parameters. To write logs to syslog,
    you need to set `log.to` to `syslog`.
]])
```